build.xml

```diff
@@ -862,7 +857,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
         <fileset dir="${conf.dir}" excludes="**/*.template"/>
       </copy>
 
-      <copy todir="${dist.dir}/docs">
+      <copy todir="${dist.dir}/zookeeper-docs">
```

Comment:
Since paths has been modified, you also need to edit releaseaudit target, excludes.
See here which files are not excluded:
https://builds.apache.org/job/PreCommit-ZOOKEEPER-github-pr-build/3290/artifact/patchprocess/patchReleaseAuditProblems.txt